apps/public-reference/pages/listing.tsx

```diff
@@ -292,17 +292,19 @@ export default class extends Component<ListingProps> {
               </div>
             </ListingDetailItem>
 
-            <ListingDetailItem
-              imageAlt="neighborhood-buildings"
-              imageSrc="/images/listing-neighborhood.svg"
-              title={t("listings.sections.neighborhoodTitle")}
-              subtitle={t("listings.sections.neighborhoodSubtitle")}
-              desktopClass="bg-primary-lighter"
-            >
-              <div className="listing-detail-panel">
-                <ListingMap address={listing.buildingAddress} listing={listing} />
-              </div>
-            </ListingDetailItem>
+            {listing?.buildingAddress.latitude && listing?.buildingAddress.longitude && (
+              <ListingDetailItem
+                imageAlt="neighborhood-buildings"
```

Comment:
Unrelated to this PR directly, but shouldn't alt tags be translatable strings, @slowbot?

Reply:
Ok, I split this off into #547, since it's really a separate thing are there are other instances as well.